F OpenNebula/one#6373: Update docs (#318)
OpenNebula/website: content/product/cluster_configuration/storage_system/local_ds.md

Just make sure that there is enough space under `/var/lib/one/datastores` to store the disks of running VMs on that Host.

{{< alert title="Warning" color="warning" >}}

> Make sure all the Hosts, including the Front-end, can SSH to any other Host (including themselves), otherwise migrations will not work.{{< /alert >}}

Reviewer: ```suggestion
Ensure that all Hosts, including the Front-end, can remotely connect via SSH to any other Host (and themselves);   otherwise, migrations will not perform as expected.{{< /alert >}}
```

```

{{< alert title="Note" color="success" >}}

> When different System Datastores are available, the `TM_MAD_SYSTEM` attribute will be set after selecting the Datastore.{{< /alert >}}

Reviewer: ```suggestion
When different System Datastores are available, select one to have the the `TM_MAD_SYSTEM` attribute set.{{< /alert >}}
```

Also note that there are additional attributes that can be set. Check the [datastore template attributes]({{% relref "datastores#datastore-common" %}}).

{{< alert title="Warning" color="warning" >}}
Be sure to use the same `TM_MAD` for both the System and Image datastores. When combining different transfer modes, check the section below.{{< /alert >}}

### Additional Configuration

> - `DD_BLOCK_SIZE`: Block size for dd operations (default: 64kB). Can be set in `/var/lib/one/remotes/etc/datastore/fs/fs.conf`.

Reviewer: ```suggestion
- `DD_BLOCK_SIZE`: Block size for dd operations (default: 64kB). Configured in `/var/lib/one/remotes/etc/datastore/fs/fs.conf`.
```

- `SUPPORTED_FS`: Comma-separated list of every filesystem supported for creating formatted datablocks. Can be set in `/var/lib/one/remotes/etc/datastore/datastore.conf`.

> - `FS_OPTS_<FS>`: Options for creating the filesystem for formatted datablocks. Can be set in `/var/lib/one/remotes/etc/datastore/datastore.conf` for each filesystem type.

Reviewer: ```suggestion
- `FS_OPTS_<FS>`: Options for creating the filesystem for formatted datablocks. Configured in `/var/lib/one/remotes/etc/datastore/datastore.conf` for each filesystem type.
```
